<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Expression_evaluationClass</class>
 <widget class="QMainWindow" name="Expression_evaluationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>524</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>524</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>524</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>计算器</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>保留小数点后4位</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="3">
       <widget class="QPushButton" name="pushButton_14">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(240, 240, 240, 100);
background-color: rgb(255, 255, 255);
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qt_calculator.qrc">
          <normaloff>:/Qt_calculator/Resources/d4628535e5dde711b23510aea6efce1b9c166186.png</normaloff>:/Qt_calculator/Resources/d4628535e5dde711b23510aea6efce1b9c166186.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_18">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(240, 240, 240, 100);
background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qt_calculator.qrc">
          <normaloff>:/Qt_calculator/Resources/QQ图片20170921182812.png</normaloff>:/Qt_calculator/Resources/QQ图片20170921182812.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>45</width>
          <height>45</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_5">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(240, 240, 240, 100);
background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qt_calculator.qrc">
          <normaloff>:/Qt_calculator/Resources/5_48px_1168277_easyicon.net - 副本.png</normaloff>:/Qt_calculator/Resources/5_48px_1168277_easyicon.net - 副本.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>58</width>
          <height>58</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButton_6">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(240, 240, 240, 100);
background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qt_calculator.qrc">
          <normaloff>:/Qt_calculator/Resources/6_48px_1168279_easyicon.net.png</normaloff>:/Qt_calculator/Resources/6_48px_1168279_easyicon.net.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>58</width>
          <height>58</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_17">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(240, 240, 240, 100);
background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qt_calculator.qrc">
          <normaloff>:/Qt_calculator/Resources/QQ图片20170921182800.png</normaloff>:/Qt_calculator/Resources/QQ图片20170921182800.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>45</width>
          <height>45</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="pushButton_13">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(240, 240, 240, 100);
background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qt_calculator.qrc">
          <normaloff>:/Qt_calculator/Resources/little-minus-symbol_318-25368.png</normaloff>:/Qt_calculator/Resources/little-minus-symbol_318-25368.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButton_1">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(240, 240, 240, 100);
background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qt_calculator.qrc">
          <normaloff>:/Qt_calculator/Resources/1_48px_1168269_easyicon.net - 副本.png</normaloff>:/Qt_calculator/Resources/1_48px_1168269_easyicon.net - 副本.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>58</width>
          <height>58</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pushButton_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(240, 240, 240, 100);
background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qt_calculator.qrc">
          <normaloff>:/Qt_calculator/Resources/2_48px_1168271_easyicon.net - 副本.png</normaloff>:/Qt_calculator/Resources/2_48px_1168271_easyicon.net - 副本.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>58</width>
          <height>58</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="pushButton_3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(240, 240, 240, 100);
background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qt_calculator.qrc">
          <normaloff>:/Qt_calculator/Resources/3_48px_1168273_easyicon.net.png</normaloff>:/Qt_calculator/Resources/3_48px_1168273_easyicon.net.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>58</width>
          <height>58</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="pushButton_12">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(240, 240, 240, 100);
background-color: rgb(255, 255, 255);
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qt_calculator.qrc">
          <normaloff>:/Qt_calculator/Resources/a08b87d6277f9e2fda25102e1d30e924b899f380 - 副本.png</normaloff>:/Qt_calculator/Resources/a08b87d6277f9e2fda25102e1d30e924b899f380 - 副本.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_9">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(240, 240, 240, 100);
background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qt_calculator.qrc">
          <normaloff>:/Qt_calculator/Resources/9_48px_1168285_easyicon.net.png</normaloff>:/Qt_calculator/Resources/9_48px_1168285_easyicon.net.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>58</width>
          <height>58</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton_16">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(240, 240, 240, 100);
background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qt_calculator.qrc">
          <normaloff>:/Qt_calculator/Resources/delete_point_26px_1073731_easyicon.net.png</normaloff>:/Qt_calculator/Resources/delete_point_26px_1073731_easyicon.net.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>58</width>
          <height>58</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_8">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(240, 240, 240, 100);
background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qt_calculator.qrc">
          <normaloff>:/Qt_calculator/Resources/8_48px_1168283_easyicon.net - 副本.png</normaloff>:/Qt_calculator/Resources/8_48px_1168283_easyicon.net - 副本.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>58</width>
          <height>58</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pushButton_15">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(240, 240, 240, 100);
background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qt_calculator.qrc">
          <normaloff>:/Qt_calculator/Resources/a1ec08fa513d269721a8857457fbb2fb4316d8a4.png</normaloff>:/Qt_calculator/Resources/a1ec08fa513d269721a8857457fbb2fb4316d8a4.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_7">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(240, 240, 240, 100);
background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qt_calculator.qrc">
          <normaloff>:/Qt_calculator/Resources/7_48px_1168281_easyicon.net.png</normaloff>:/Qt_calculator/Resources/7_48px_1168281_easyicon.net.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>58</width>
          <height>58</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="pushButton_11">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(240, 240, 240, 100);
background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qt_calculator.qrc">
          <normaloff>:/Qt_calculator/Resources/20140429190312-1688988626.png.jpg</normaloff>:/Qt_calculator/Resources/20140429190312-1688988626.png.jpg</iconset>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="pushButton_19">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(240, 240, 240, 100);
background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qt_calculator.qrc">
          <normaloff>:/Qt_calculator/Resources/t0133292e59531bace9.png</normaloff>:/Qt_calculator/Resources/t0133292e59531bace9.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="pushButton_0">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(240, 240, 240, 100);
background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qt_calculator.qrc">
          <normaloff>:/Qt_calculator/Resources/0_48px_1168267_easyicon.net.png</normaloff>:/Qt_calculator/Resources/0_48px_1168267_easyicon.net.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>58</width>
          <height>58</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="pushButton_10">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(240, 240, 240, 100);
background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qt_calculator.qrc">
          <normaloff>:/Qt_calculator/Resources/QQ图片20170921184942.png</normaloff>:/Qt_calculator/Resources/QQ图片20170921184942.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_4">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(240, 240, 240, 100);
background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qt_calculator.qrc">
          <normaloff>:/Qt_calculator/Resources/4_48px_1168275_easyicon.net.png</normaloff>:/Qt_calculator/Resources/4_48px_1168275_easyicon.net.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>58</width>
          <height>58</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>350</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qt_calculator.qrc"/>
 </resources>
 <connections/>
</ui>
